common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -3831,6 +3831,12 @@
     ],
     "sqlState" : "42K08"
   },
+  "SHOW_COLUMNS_WITH_CONFLICT_NAMESPACE" : {
```

Comment:
I'm not whether it's a good practice or not to create a class/category name that nearly matches the detailed content it holds

Reply:
In the file `error-conditions.json`, there is a similar one
https://github.com/apache/spark/blob/ac5aa7dcb0a5a85cf05a334912305abeda11f6c2/common/utils/src/main/resources/error/error-conditions.json#L704-L709

Reply:
That's bad. Users almost read the same content twice in a single error.